server/container/cli/src/main/java/org/apache/james/cli/ServerCmd.java

```diff
@@ -125,18 +136,6 @@ public static void executeAndOutputToStream(String[] args, PrintStream printStre
             printStream);
         stopWatch.stop();
     }
-
-    private final DataProbe probe;
-    private final MailboxProbe mailboxProbe;
-    private final QuotaProbe quotaProbe;
-    private final SieveProbe sieveProbe;
-    
-    public ServerCmd(DataProbe probe, MailboxProbe mailboxProbe, QuotaProbe quotaProbe, SieveProbe sieveProbe) {
-        this.probe = probe;
-        this.mailboxProbe = mailboxProbe;
-        this.quotaProbe = quotaProbe;
-        this.sieveProbe = sieveProbe;
-    }
```

Comment:
That is unfortunate: we tend to put static methods first, then fields, then the constructor, then the methods.

You also moved by mistake in a similar way `print` and `printUsage` below.

Could you restore the original ordering of these methods?

Reply:
I did a Rearrange Code in Intellij which i thought used the checkstyle to rearrange the code accordingly. Reverting it :)